### ISSUE TITLE
Cannot read property 'document' of undefined

### ISSUE THREAD

**Issue by vlazh** [zabel-xyz/local-history#31]
- Extension Name: local-history
- Extension Version: 1.6.2
- OS Version: Linux x64 4.10.0-42-generic
- VSCode version: 1.19.0-insider

```json 
{
	"activationTimes": {
		"startup": true,
		"codeLoadingTime": 62,
		"activateCallTime": 1,
		"activateResolvedTime": 0,
		"activationEvent": "*"
	},
	"runtimeErrors": [
		{
			"name": "TypeError",
			"message": "Cannot read property 'document' of undefined"
		}
	]
}
```

In console:
```
TypeError: Cannot read property 'document' of undefined
	at HistoryTreeProvider.changeActiveFile (.../.vscode-insiders/extensions/xyz.local-history-1.6.2/out/src/historyTree.provider.js:149:56)
	at vscode.window.onDidChangeActiveTextEditor.e (.../.vscode-insiders/extensions/xyz.local-history-1.6.2/out/src/extension.js:50:65)
	at e.fire (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:87:218)
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:485:858
	at e.fire (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:87:218)
	at e.$acceptDocumentsAndEditorsDelta (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:435:723)
	at t.e.invoke (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:516:87)
	at e._invokeHandler (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:357:580)
	at e._receiveOneMessage (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:357:270)
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:356:206
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:358:204
	at /usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:85:988
	at e.fire (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:87:218)
	at Socket.<anonymous> (/usr/share/code-insiders/resources/app/out/vs/workbench/node/extensionHostProcess.js:135:717)
	at emitOne (events.js:96:13)
	at Socket.emit (events.js:191:7)
```

**Comment by zabel-xyz** [comment]
Special editor like welcome, html preview... trigger event onDidChangeActiveTextEditor() but activeTextEdtior is null in this case.

I'll fix it...

**Comment by zabel-xyz** [comment]
Fix in version 1.7.0
